functional_tests/login/03-login-cloud-local-key.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+"""WB-8618: warn the user if they are using a local key to login to cloud"""
+
+import pytest
+import wandb
+import wandb.errors
+
+
+if __name__ == "__main__":
+    with pytest.raises(wandb.errors.UsageError) as e:
+        wandb.login(key="local1234567890")
+        assert (
+            "Attempting to use a local API key to connect to https://api.wandb.ai" in str(e.value)
+        )
```

Comment:
just curious why not use a unit-test for this case?

Reply:
oh, quite simple: it's laziness 🤣 
i initially wrote a unit test that was failing in mysterious ways in CI in certain environments and I was to lazy to figure out what needs to be mocked etc. to make it work there.
Also, a yea test looks so clean vs a unit test with all those mocks and fixtures -- tests exactly what should be tested.

Reply:
i wonder if we could use this as a model to move everything to yea? it will solve our problem with the mock server.
something to think about (we could keep strictly unit-tests in pytest)